rerun_py/rerun_sdk/rerun/datatypes/time_int_ext.py

```diff
@@ -10,26 +10,31 @@ def __init__(self: Any, *, seq: int | None = None, seconds: float | None = None,
         """
         Create a new instance of the TimeInt datatype.
 
+        Exactly one of `seq`, `seconds`, or `nanos` must be provided.
```

Comment:
I think we should be able to roll the `@overload` stuff on `__init__` here too.

Reply:
I'd rather not take the time to do that now. That's an orthogonal improvement